workflow/artifacts/http/http_test.go

```diff
@@ -1,17 +1,58 @@
 package http
 
 import (
+	"bytes"
 	"os"
+	"regexp"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	log "github.com/sirupsen/logrus"
 
 	"github.com/argoproj/argo/errors"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
goimports

Reply:
Sorry, what do you mean?

Reply:
Seems like this file's imports are correct according to my IDE at least.

Reply:
try removing all the blank lines and then re-formatting the imports

Reply:
Done!